pipeline_documentation.md

```diff
@@ -87,6 +94,7 @@ on installation and usage please see [here](README.md).
 
 | Name | License | Tag line | More info |
 | --- | --- | --- | --- |
+| **ASCII-style alignment pileups** | [Apache 2.0][license-apache2] | _"Generates an ASCII-style pileup of read alignments in one or more BAM files against one or more regions specified in a BED file."_ | [code][code-ascii] | 
```

Comment:
I think the description is a bit inaccurate:
- `Generates an`: I think the script can generate multiple pileups in one run, no? So should be plural.
- `against`: I think "for" is actually more accurate :)
- `regions specified in a BED file`: Maybe not so interesting here that it requires regions in a BED format. Rather I would just say "for one or more genomic regions."

Reply:
Changing it, but just for you to know, this is exactly what's on GitLab as the tool's description.